backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/stubs.kt

```diff
@@ -20,6 +20,7 @@ abstract class Stub<out D : DeclarationDescriptor>(val name: String, val comment
         get() = ((descriptor as? DeclarationDescriptorWithSource)?.source as? PsiSourceElement)?.psi
     open val isValid: Boolean
         get() = descriptor?.module?.isValid ?: true
+    internal open val shouldExportKDoc = false
```

Comment:
Adding this property doesn't make much sense if you take the flag as a parameter to `render`.
Also associated refactorings shouldn't be required either.

Reply:
I mean that it is enough to pass `shouldExportKDoc` to `render()` from `ObjCExportHeaderGenerator` instead.